clientbuffer.cpp

```diff
@@ -279,16 +289,15 @@ timeval CClientBufferMod::GetTimestamp(const CBuffer& buffer) const
 bool CClientBufferMod::HasSeenTimestamp(const CString& identifier, const CString& target, const timeval& tv)
 {
     const timeval seen = GetTimestamp(identifier, target);
-    return timercmp(&seen, &tv, >);
+    return timercmp(&seen, &tv, >=);
```

Comment:
According to `man timercmp`, this might be unportable. Consider instead `return !timercmp(&seen, &tv, <);`